backend/src/apiserver/server/run_server.go

```diff
@@ -134,6 +134,14 @@ func (s *RunServer) validateCreateRunRequest(request *api.CreateRunRequest) erro
 	return nil
 }
 
+func (s *RunServer) TerminateRun(ctx context.Context, request *api.TerminateRunRequest) (*empty.Empty, error) {
+	err := s.resourceManager.TerminateRun(request.RunId)
+	if err != nil {
```

Comment:
Replace this method's body with just a call to TerminateRun?
```suggestion
	return nil, s.resourceManager.TerminateRun(request.RunId)
```

Reply:
I think the success return value needs to be `return &empty.Empty{}, nil`, not `nil, nil`

Reply:
That's the reason, yes.